etl/storage/storage.go

```diff
@@ -229,3 +229,27 @@ func (s *storage) GetJetDrops(pulse models.Pulse) ([]models.JetDrop, error) {
 	err := s.db.Where("pulse_number = ?", pulse.PulseNumber).Find(&jetDrops).Error
 	return jetDrops, err
 }
+
+func (s *storage) GetJetDropsWithParams(pulse models.Pulse, fromJetDropID *string, limit int, offset int) ([]models.JetDrop, int, error) {
```

Comment:
Why function gets `models.Pulse`, if we only use pulseNumber inside and have `int` value to call it?

Reply:
love to have typed params instead of "int", when it possible